CODEOWNERS

```diff
@@ -1,4 +1,4 @@
-# CODEOWNERS for autoreview assigning in github
+# CODEOWNERS for autoreview assigning in githubbmd
```

Comment:
is this intentional @brec-u-blox ?

Reply:
ugh - This must have crept in when searching in nano and thought I hit ^w first. I'll fix.